README.md

```diff
@@ -1,10 +1,12 @@
-# Metriport HAPI-FHIR Server
+# Metriport FHIR Server
 
 This FHIR server is a fork of https://github.com/hapifhir/hapi-fhir-jpaserver-starter.
 
-Note that this project is specifically intended for end users of the HAPI FHIR JPA server module (in other words, it helps you implement HAPI FHIR, it is not the source of the library itself). If you are looking for the main HAPI FHIR project, see here: https://github.com/hapifhir/hapi-fhir
-
-Need Help? Please see: https://github.com/hapifhir/hapi-fhir/wiki/Getting-Help
+The main difference from the original repository is Auth* - this is based on [Metriport](https://github.com/metriport/metriport)'s OAuth2:
```

Comment:
typo in `Auth*`?

Reply:
I was referring to both AuthN (Authentication) and AuthZ (Authorization). Will move to the long version.